.gitlab-ci.yml

```diff
@@ -1,9 +1,70 @@
 stages:
+  - test
+  - build image
+  - trigger deploy
   - publish-docs-if-changed
   - veracode scan
   - deps scan
   - generate pages
 
+variables:
+  VAULT_ADDR: "http://vault.sit-ota.aws.in.here.com"
+
+test:
+  stage: test
+  tags:
+    - docker-sock
+  image: advancedtelematic/gitlab-jobs:0.2.3
+  script:
+    - sbt -sbt-dir ./.sbt -ivy ./.ivy2 -J-Xmx1G -J-XX:ReservedCodeCacheSize=128m ota-plus-web/ut:test
```

Comment:
Lets use `sbt -sbt-dir ./.sbt -ivy ./.ivy2 ota-plus-web/ut:test`, if it works. In general we should not limit the memory anymore, I think this should work.